src/geometry/manifold/instances/units_of_normed_algebra.lean

```diff
@@ -0,0 +1,40 @@
+/-
+Copyright © 2021 Heather Macbeth, Nicolò Cavalleri. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Heather Macbeth, Nicolò Cavalleri
+-/
+
+import geometry.manifold.algebra.lie_group
+import analysis.normed_space.units
+
+/-!
+# Units of normed algebra
+-/
+
+noncomputable theory
+
+instance units_of_normed_algebra.charted_space {𝕜 : Type*} [nondiscrete_normed_field 𝕜]
+  {R : Type*} [normed_ring R] [normed_algebra 𝕜 R] [complete_space R] :
+  charted_space R (units R) :=
+units.open_embedding_coe.charted_space
+
+namespace units
+
+variables {𝕜 : Type*} [nondiscrete_normed_field 𝕜]
+variables {R : Type*} [normed_ring R] [normed_algebra 𝕜 R] [complete_space R]
+
+/- Why does it not find the 𝕜 alone? -/
+@[simp, mfld_simps] lemma chart_at.apply {a : units R} {b : units R} :
+  @chart_at R _ _ _ (@units_of_normed_algebra.charted_space 𝕜 _ _ _ _ _) a b = b := rfl
```

Comment:
Does anybody know anything aout this?

Reply:
I would guess that Lean would fill it by itself if I pass `𝕜` as an explicit argument to the lemma though, which does not happen...

Reply:
Oh actually, I think the point is that you should not have a `𝕜` in the charted space construction.  It works for any normed ring, the `𝕜`-algebra structure isn't being used until the smooth manifold construction.